react/SquareAppIcon/styles.styl

```diff
@@ -55,7 +55,77 @@ $color = constants['color'] // hard-coded color, because the component is curren
         transform scale(0.8334)
 
 .SquareAppIcon-spinner
+    margin 0 !important
+
     svg
         position absolute
         height 100%
         width 100%
+
+animation-duration = 1.50s
```

Comment:
I'm not used to stylus files. Is that an equivalent to css `var()` ? If yes, is this scoped to this file?

Reply:
they will be local to that file, they are not equivalent to CSS var but more like sass variables. I didn't feel the need to have them as CSS vars because I don't have to change them at runtime (animationColor though, needs to be changed at runtime and thus is a CSS var)